san/batch.py

```diff
@@ -21,11 +22,12 @@ def __create_batched_query_string(self):
         batched_queries = []
 
         for idx, query in enumerate(self.queries):
-            [metric, slug] = query[0].split('/')
+            [metric, _separator, slug] = query[0].partition('/')
```

Comment:
why change to `partition` when you ignore the separator?

Reply:
Because of split's nature - when we do [metric, slug] = ...split('/'), and there isn't a '/', then there isn't a value for slug, whereas partition just makes slug = ''